src/main.ts

```diff
@@ -32,17 +33,31 @@ import { isNotFoundError } from './utils/http'
 
 const app = createApp(App)
 const pinia = createPinia()
-const i18n = setupI18n()
 const head = createHead()
 
 routerPromise
   .then((router) => {
     app.use(router)
     app.use(pinia)
-    app.use(i18n)
     app.use(head)
     app.use(TextClamp)
     app.use(LoadingPlugin)
+    app.use(datagouv, {
+      name: 'data.gouv.fr',
+      baseUrl: config.datagouvfr.base_url,
+      apiBase: config.datagouvfr.base_url,
+      staticUrl: 'https://static.data.gouv.fr',
+      tabularApiUrl: 'https://tabular-api.data.gouv.fr',
```

Comment:
There's a brand new setting for that https://github.com/opendatateam/udata-front-kit/blob/d22bd65f275f94f7c15a4fac0f6e5be66c50308f/configs/ecospheres/config.yaml#L11